Sources/SpeziAccessGuard/CodeViews/CodeView.swift

```diff
@@ -63,11 +66,27 @@ struct CodeView: View {
                 oldKeyBoardType = codeOption.keyBoardType
             }
             .onChange(of: code) {
-                if code.count >= codeOption.maxLength {
-                    Task { @MainActor in
-                        await checkCode()
+                ///Validation is skipped, when code was reset or modified after an error
```

Comment:
Inline comments generally only use two `/`s, documentation comments to be rendered in documentation use `///`: https://www.swift.org/documentation/docc/writing-symbol-documentation-in-your-source-files
```suggestion
                // Validation is skipped, when code was reset or modified after an error
```